indico/modules/vc/models/vc_rooms.py

```diff
@@ -337,6 +337,7 @@ def delete(self, user, delete_all = 0):
         If the room is not used anywhere else, the room itself is also deleted.
 
         :param user: the user performing the deletion
+        :param delete_all: if set to 1 all events linked to a videoconference room will be deleted together with the room. Default set to false.
```

Comment:
This line is way too long - text/comment lines should be max 72 chars. Also, the doc is wrong. no *events* are deleted... I think this would be clearer:

```
        :param delete_all: if True, the room is detached from all 
                           events and deleted.
```